java/src/main/java/com/saucelabs/saucebindings/VisualTestingStrategy.java

```diff
@@ -0,0 +1,48 @@
+package com.saucelabs.saucebindings;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.openqa.selenium.InvalidArgumentException;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class VisualTestingStrategy implements APIStrategy {
+    private final SauceOptions sauceOptions;
+
+    public VisualTestingStrategy(SauceOptions options) {
+        sauceOptions = options;
+    }
+    @Getter
+    private RemoteWebDriver driver;
+    @Setter
+    public URL sauceUrl;
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/SauceLabs/sauce_bindings/pullRequest?prid=6386972)